test/Android/androidtest.cmd

```diff
@@ -34,6 +35,14 @@ set _GlassDir=%_ProjectRoot%%_GlassPackageName%\
 if NOT exist "%_GlassDir%glass2.exe" echo Getting Glass from NuGet.& call "%_ProjectRoot%tools\NuGet\nuget.exe" install %_GlassPackageName% -Version %_GlassPackageVersion% -ExcludeVersion -OutputDirectory %_ProjectRoot%
 if NOT "%ERRORLEVEL%"=="0" echo ERROR: Failed to get Glass from NuGet.& exit /b -1
 
+:: Copy binaries to folder "Microsoft.VisualStudio.Glass" which required by glass2.exe in runtime for Visual Studio 2017
+if defined VS150COMNTOOLS (
+xcopy /Y /D "%VSINSTALLDIR%Common7\IDE\Remote Debugger\x86\Microsoft.VisualStudio.Debugger.Interop.15.0.dll" "%_GlassDir%"
```

Comment:
Are you running the glass tests from an admin command prompt? Maybe try deleting HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Glass (note: this is a **32** bit registry key, so if you are using 64-bit regedit, look under HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node).

The version of glass that AndroidTest.cmd pulls is Dev14-based. So it definitely shouldn't need the 15 interop assembly.